docs/proposals/202105256-cluster-class-and-managed-topologies.md

```diff
@@ -253,6 +253,9 @@ type LocalObjectTemplate struct {
       Metadata ObjectMeta `json:"metadata,omitempty"`
     
       // The number of control plane nodes.
+      // If the value is nil, the ControlPlane object is created without the number of Replicas
+      // and it's assumed that the control plane controller does not implement support for this field.
+      // +optional
```

Comment:
Same as above comment around ignoring unsupported Replicas field in Control Plane providers.